pkg/client/cli/cmd_status.go

```diff
@@ -113,7 +113,7 @@ func connectorStatus(cmd *cobra.Command) error {
 
 		if !cliutil.HasLoggedIn(ctx) {
 			fields = append(fields, kv{"Ambassador Cloud", "Logged out"})
-		} else if _, err := cliutil.GetCloudAccessToken(ctx, false); err != nil {
+		} else if _, err := cliutil.GetCloudUserInfo(ctx, false, true); err != nil {
 			fields = append(fields, kv{"Ambassador Cloud", "Login expired"})
```

Comment:
This assumes that the only error returned from the `GetCloudUserInfo()` is that the login expired. Other errors will go undetected.

Reply:
That's pretty dang close to being true.  "We've previously logged in, but the login is no longer functioning"